addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/InformationDisclosureDebugErrorsScanRuleUnitTest.java

```diff
@@ -157,7 +158,7 @@ void alertsIfDebugErrorsDisclosed(String debugError) throws HttpMalformedHeaderE
         assertThat(alert.getUri(), equalTo(URI));
         assertThat(alert.getRisk(), equalTo(Alert.RISK_LOW));
         assertThat(alert.getConfidence(), equalTo(Alert.CONFIDENCE_MEDIUM));
-        assertThat(alert.getCweId(), equalTo(200));
+        assertThat(alert.getCweId(), equalTo(1295));
         assertThat(alert.getWascId(), equalTo(13));
```

Comment:
Definitely asking for a method to be extracted, or just removed, that we are checking in the example should be more than enough.